### DISCUSSION TITLE
Hosted SPF and DKIM

### DISCUSSION THREAD

**Discussion by DirectITServicesUK** [postalserver/postal#3052] [Development discussions]
Not really a bug? I host my DKIM, SPF and other DNS security settings with PowerDMARC.
My SPF record,  "v=spf1 include:42b3kgsb2s.powerspf.com ~all",  and the hosted service provides all the settings.
Postal shows this.
![image](https://github.com/user-attachments/assets/c5bb7aaf-6590-4150-9957-b8562633defd)

My SPF set up is 
![image](https://github.com/user-attachments/assets/072e9ad1-ea2d-470b-b3a1-b0963e2003e7)

Emails are delivered fine though.

 - OS: Ubuntu 22 - Docker Setup
 - Version 3.3.4

**Comment by willpower232** [comment]
related #3029
